### PR TITLE
fixed readme

### PR DESCRIPTION
**Pull request by DevanshXerc90**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,47 +1,75 @@
...
 
-### 6. Gallery:
-- Share your astrophotography images, sketches, and artwork.
-- Explore a diverse collection of celestial images contributed by members of the community.
+To contribute to the APS Website, please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for detailed instructions.
 
-### 7. Educational Outreach:
-- Get involved in outreach activities aimed at promoting astronomy education in schools and communities.
-- Volunteer for events such as public stargazing sessions, school presentations, etc.
+## License
```
Comment (singh-ab) [inline README.md:65]: You've added license reference but there is no license file. Remove license reference for now.

File: README.md [hunk 2]
```diff
@@ -1,47 +1,75 @@
...
+
+- **Join Us**: Interested in astronomy and physics? Join our society effortlessly through our website. We welcome students and enthusiasts who share our passion for exploring the cosmos.
+
+- **Event Details**: Get comprehensive information about upcoming and past events, including schedules, topics, and participation details. We aim to provide engaging and educational experiences for all members.
+
+- **Gallery**: Explore our gallery featuring photos and highlights from our past events. From stargazing nights to guest lectures, relive the moments with us.
+
+- **Educational Outreach**: Engage in activities aimed at promoting astronomy education in schools and communities. Volunteer for events such as public stargazing sessions and school presentations, and contribute to fostering a love for astronomy among young minds.
+
+## Connect with Us
```
Comment (singh-ab) [inline README.md:19]: Connect and contact section are overlapping.
Why did you add Hacktoberfest logo again?

Comment (singh-ab) [reply README.md:19]: Remove this section altogether

File: README.md [hunk 3]
```diff
@@ -1,47 +1,75 @@
...
-### 7. Educational Outreach:
-- Get involved in outreach activities aimed at promoting astronomy education in schools and communities.
-- Volunteer for events such as public stargazing sessions, school presentations, etc.
+## License
 
-## How to contribute:
+Distributed under the MIT License. See [LICENSE](LICENSE) for more information.
 
-To contribute to the APS Website, please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for detailed instructions.
+## Contact
```
Comment (singh-ab) [inline README.md:69]: Just write the mail address, i.e. astronomy@iiitdmj.ac.in
Remove other contact since it will be on the github org page.

File: README.md [hunk 4]
```diff
@@ -1,47 +1,75 @@
...
+
+- **Member Information**: Learn about the passionate individuals behind APS. Our members are the driving force of our society, and we take pride in their dedication and contributions.
+
+- **Join Us**: Interested in astronomy and physics? Join our society effortlessly through our website. We welcome students and enthusiasts who share our passion for exploring the cosmos.
+
+- **Event Details**: Get comprehensive information about upcoming and past events, including schedules, topics, and participation details. We aim to provide engaging and educational experiences for all members.
+
+- **Gallery**: Explore our gallery featuring photos and highlights from our past events. From stargazing nights to guest lectures, relive the moments with us.
+
+- **Educational Outreach**: Engage in activities aimed at promoting astronomy education in schools and communities. Volunteer for events such as public stargazing sessions and school presentations, and contribute to fostering a love for astronomy among young minds.
```
Comment (singh-ab) [inline README.md:17]: Volunteer is good but Outreach also represents the outreach events we offer or do.

**Review by singh-ab — CHANGES_REQUESTED** [review]

**Review by DevanshXerc90 — COMMENTED** [review]
fixed
